### ISSUE TITLE
Napari GUI not displaying properly on OSX

### ISSUE THREAD

**Issue by marshuang80** [napari/napari#1044]
## 🐛 Bug

The buttons and labels are not displayed properly on the Napari GUI so it's hard to tell the function of each button (e.g. the buttons don't have proper labels). Scaling seem a bit off as well. 

## To Reproduce

1. Download Napari by ```pip install git+https://github.com/napari/napari```
2. Open Napari GUI

## Expected behavior

Napari GUI that has buttons with labels on to indicate the functions of these buttons. 

## Environment

 - Please copy and paste the information at napari info option in help menubar here:

Napari Version: "0.2.12+13.g546d138
OXS Version: 10.15.2
Python Version: 3.7.6

## Additional context

![image](https://user-images.githubusercontent.com/15271135/76464332-60e9ac80-63a2-11ea-86e1-d6c78b260d8d.png)

**Comment by tlambert03** [comment]
Hi @marshuang80, sorry about that.  
There are some known issues at the moment with installing directly from the master branch as you have done here (vs using `pip install napari`), it is being fixed in #1037.  I suspect, but would like to know for sure, that you are using an older version of Qt?  Can you confirm that for me?
use `napari --info` at the command line, or use `pip freeze` (or `conda list` if you're in a conda env) and paste the output here.

quickest solution for you is to install from the latest release rather than the github master branch
```
pip uninstall napari -y
pip install napari
```

**Comment by tlambert03** [comment]
This is a duplicate of https://github.com/napari/napari/issues/1032

**Comment by marshuang80** [comment]
Hey @tlambert03, 

Thank you for your reply. The follow is the output of ```napari --info```

```
napari: 0.2.12+13.g546d138
Platform: Darwin-19.2.0-x86_64-i386-64bit
Python: 3.7.6 (default, Dec 30 2019, 19:38:26)  [Clang 11.0.0 (clang-1100.0.33.16)]
Qt: 5.14.1
PySide2: 5.14.1
VisPy: 0.6.4
NumPy: 1.18.1
SciPy: 1.4.1
scikit-image: 0.16.2
Dask: 2.12.0
GL version:  2.1 ATI-3.4.19
MAX_TEXTURE_SIZE: 16384
```
Additionally, I am getting a ```AssertionError: Multiple .dist-into directorises``` when I try to install napari via ```pip install napari``.  This is probably a local issue so I don't think I will need to open a seperate issue.

**Comment by tlambert03** [comment]
Oh very strange.  do you get any other errors or warnings when you start up napari?  I haven't seen this problem with that version of `Qt`...  Can you provide the full output of `pip freeze`?  I'm curious to see your environment.  I'll try to reproduce it here.

The pip install issue probably means that you have an older version of pip installed.  That is described in https://github.com/napari/napari/issues/1009, and fixed in https://github.com/napari/napari/pull/1011 (so it shouldn't be an issue in the next napari release, or if you update pip).

**Comment by marshuang80** [comment]
Updating my pip did solve the installation problem, thank you so much!

I didn't install napari in an environment so there is a lot of packages in ```pip freeze``. Here is the full list: 

alabaster==0.7.12
anaconda-client==1.7.2
anaconda-navigator==1.9.7
anaconda-project==0.8.3
appnope==0.1.0
appscript==1.0.1
asn1crypto==1.0.1
astroid==2.3.1
astropy==3.2.2
atomicwrites==1.3.0
attrs==19.2.0
Babel==2.7.0
backcall==0.1.0
backports.functools-lru-cache==1.5
backports.os==0.1.1
backports.shutil-get-terminal-size==1.0.0
backports.tempfile==1.0
backports.weakref==1.0.post1
beautifulsoup4==4.8.0
bitarray==1.0.1
bkcharts==0.2
bleach==3.1.0
bokeh==1.3.4
boto==2.49.0
Bottleneck==1.2.1
certifi==2019.9.11
cffi==1.12.3
chardet==3.0.4
Click==7.0
cloudpickle==1.2.2
clyent==1.2.2
colorama==0.4.1
conda==4.8.1
conda-build==3.18.9
conda-package-handling==1.6.0
conda-verify==3.4.2
contextlib2==0.6.0
cryptography==2.7
cycler==0.10.0
Cython==0.29.13
cytoolz==0.10.0
dask==2.5.2
decorator==4.4.0
defusedxml==0.6.0
distributed==2.5.2
docutils==0.15.2
entrypoints==0.3
et-xmlfile==1.0.1
fastcache==1.1.0
filelock==3.0.12
Flask==1.1.1
freetype-py==2.1.0.post1
fsspec==0.5.2
future==0.17.1
gevent==1.4.0
glob2==0.7
gmpy2==2.0.8
greenlet==0.4.15
h5py==2.9.0
HeapDict==1.0.1
html5lib==1.0.1
idna==2.8
imageio==2.6.0
imagesize==1.1.0
importlib-metadata==0.23
ipykernel==5.1.2
ipython==7.8.0
ipython-genutils==0.2.0
ipywidgets==7.5.1
isort==4.3.21
itsdangerous==1.1.0
jdcal==1.4.1
jedi==0.15.1
Jinja2==2.10.3
joblib==0.13.2
json5==0.8.5
jsonschema==3.0.2
jupyter==1.0.0
jupyter-client==5.3.3
jupyter-console==6.0.0
jupyter-core==4.5.0
jupyterlab==1.1.4
jupyterlab-server==1.0.6
keyring==18.0.0
kiwisolver==1.1.0
lazy-object-proxy==1.4.2
libarchive-c==2.8
lief==0.9.0
llvmlite==0.29.0
locket==0.2.0
lxml==4.4.1
MarkupSafe==1.1.1
matplotlib==3.1.1
mccabe==0.6.1
mistune==0.8.4
mkl-fft==1.0.14
mkl-random==1.1.0
mkl-service==2.3.0
mock==3.0.5
more-itertools==7.2.0
mpmath==1.1.0
msgpack==0.6.1
multipledispatch==0.6.0
napari==0.2.12+14.gdec5b99
navigator-updater==0.2.1
nbconvert==5.6.0
nbformat==4.4.0
networkx==2.3
nltk==3.4.5
nose==1.3.7
notebook==6.0.1
numba==0.45.1
numexpr==2.7.0
numpy==1.17.2
numpydoc==0.9.2
olefile==0.46
openpyxl==3.0.0
packaging==19.2
pandas==0.25.1
pandocfilters==1.4.2
parso==0.5.1
partd==1.0.0
path.py==12.0.1
pathlib2==2.3.5
patsy==0.5.1
pep8==1.7.1
pexpect==4.7.0
pickleshare==0.7.5
Pillow==6.2.0
pkginfo==1.5.0.1
pluggy==0.13.0
ply==3.11
prometheus-client==0.7.1
prompt-toolkit==2.0.10
psutil==5.6.3
ptyprocess==0.6.0
py==1.8.0
pycodestyle==2.5.0
pycosat==0.6.3
pycparser==2.19
pycrypto==2.6.1
pycurl==7.43.0.3
pydicom==1.4.2
pyflakes==2.1.1
Pygments==2.4.2
pylint==2.4.2
pyodbc==4.0.27
PyOpenGL==3.1.5
pyOpenSSL==19.0.0
pyparsing==2.4.2
PyQt5==5.14.1
PyQt5-sip==12.7.1
pyrsistent==0.15.4
PySide2==5.14.1
PySocks==1.7.1
pytest==5.2.1
pytest-arraydiff==0.3
pytest-astropy==0.5.0
pytest-doctestplus==0.4.0
pytest-openfiles==0.4.0
pytest-remotedata==0.3.2
python-dateutil==2.8.0
pytz==2019.3
PyWavelets==1.0.3
PyYAML==5.1.2
pyzmq==18.1.0
QtAwesome==0.6.0
qtconsole==4.5.5
QtPy==1.9.0
requests==2.22.0
rope==0.14.0
ruamel-yaml==0.15.46
scikit-image==0.15.0
scikit-learn==0.21.3
scipy==1.3.1
seaborn==0.9.0
Send2Trash==1.5.0
shiboken2==5.14.1
simplegeneric==0.8.1
singledispatch==3.4.0.3
six==1.12.0
snowballstemmer==2.0.0
sortedcollections==1.1.2
sortedcontainers==2.1.0
soupsieve==1.9.3
Sphinx==2.2.0
sphinxcontrib-applehelp==1.0.1
sphinxcontrib-devhelp==1.0.1
sphinxcontrib-htmlhelp==1.0.2
sphinxcontrib-jsmath==1.0.1
sphinxcontrib-qthelp==1.0.2
sphinxcontrib-serializinghtml==1.1.3
sphinxcontrib-websupport==1.1.2
spyder==3.3.6
spyder-kernels==0.5.2
SQLAlchemy==1.3.9
statsmodels==0.10.1
sympy==1.4
tables==3.5.2
tblib==1.4.0
terminado==0.8.2
testpath==0.4.2
toolz==0.10.0
tornado==6.0.3
tqdm==4.36.1
traitlets==4.3.3
unicodecsv==0.14.1
urllib3==1.24.2
vispy==0.6.4
wcwidth==0.1.7
webencodings==0.5.1
Werkzeug==0.16.0
widgetsnbextension==3.5.1
wrapt==1.11.2
wurlitzer==1.0.3
xlrd==1.2.0
XlsxWriter==1.2.1
xlwings==0.15.10
xlwt==1.3.0
zict==1.0.0
zipp==0.6.0

**Comment by tlambert03** [comment]
I think that #1037 is going to fix this for you.  My suspicion is that this is due to pip installing from the github master branch (rather than using `git clone`) because some of the updated styles that just got changed a few days ago are not in the manifest.ini (but they will be once #1037 merges).

If you want to be trying out bleeding edge stuff, that's definitely great! 😀and it should _usually_ work to pip install the master branch as you did (again, that's my bad, sorry)... but cloning the repo rather than pip installing it will likely give you a better representation of what we are seeing in master... (it's only during actual releases that we double check to make sure that the pip installable version is also working well).

**Comment by sofroniewn** [comment]
This has been closed by #1037
